src/components/FastScrollPDF.tsx

```diff
@@ -11,7 +11,7 @@ import type { IFastScrollPDF } from "../types/fastScrollPDF";
 const FastScrollPDF = ({
```

Comment:
We either need to import `JSX` from React 19 and have the return value set to `JSX.Element`, or (the better solution), is to do:
```
import { FC } from "react"

const FastScrollPDF: FC<IFastScrollPDF> = ({....

```

This will ensure the arguments and return types are correct.